tshirts.cpp

```diff
@@ -14,6 +14,7 @@ char size(int cms) {
 }
 
 int main() {
+    assert(size(38) == 'M');

```

Comment:
This is sufficient to make it fail. Can you think of more conditions?
Think that a customer is entering this number. What mistakes can they make while entering the digits? How many of them can the code catch? See if you can add an assert for that.